tests/models/auto/test_tokenization_auto.py

```diff
@@ -201,6 +201,11 @@ def test_PreTrainedTokenizerFast_from_pretrained(self):
         self.assertEqual(tokenizer.padding_side, "right")
         self.assertEqual(tokenizer.truncation_side, "right")
 
+    def test_PreTrainedTokenizerFast_inferred(self):
```

Comment:
@ArthurZucker added this test for https://github.com/huggingface/transformers/pull/33751 as Siglip would be the first model that can test this (https://github.com/huggingface/transformers/pull/33751#issuecomment-2382452487)